core/util_test.go

```diff
@@ -144,6 +150,41 @@ func TestIsAnyNilOrZero(t *testing.T) {
 	test.Assert(t, !IsAnyNilOrZero(durationpb.New(666)), "A *durationpb.Duration with valid inner duration is seen as zero")
 }
 
+func BenchmarkIsAnyNilOrZero(b *testing.B) {
+	var thyme *time.Time
+	var table = []struct {
+		input interface{}
+	}{
+		{input: int(0)},
+		{input: int(1)},
+		{input: uint32(0)},
+		{input: uint64(1)},
+		{input: float32(0)},
+		{input: float32(0.1)},
+		{input: ""},
+		{input: "ahoyhoy"},
+		{input: []byte{}},
+		{input: []byte{0}},
+		{input: []byte{1}},
```

Comment:
Looks like we're missing `[]string{}`, `[]string{""}`, and `[]string{"foo"}` here, from a symmetry perspective.

Reply:
Can do.